assets/js/blocks/checkout/inner-blocks/checkout-billing-address-block/customer-address.tsx

```diff
@@ -113,26 +106,6 @@ const CustomerAddress = ( {
 					fields={ addressFieldKeys }
 					fieldConfig={ addressFieldsConfig }
 				/>
-				{ showPhoneField && (
-					<PhoneNumber
-						id="billing-phone"
-						errorId={ 'billing_phone' }
-						isRequired={ requirePhoneField }
-						value={ billingAddress.phone }
-						onChange={ ( value ) => {
-							setBillingPhone( value );
-							dispatchCheckoutEvent( 'set-phone-number', {
```

Comment:
It was not clear why those events were fired by their own (maybe because editing the field didn't trigger the parent `set-shipping-address`|`set-billing-address` hook). This is now deleted. It was experimental.